src/services/render/index.tsx

```diff
@@ -80,24 +75,27 @@ export const renderElement = (
     );
 
     if (Component) {
+      // Prepare props for the component
+      const props = {
+        element,
+        onUpdate,
+        options: {
+          ...options,
+          inlineFormattingContext,
+        },
+        stylesheets,
+      };
+
       /* eslint-disable max-len */
```

Comment:
minor: can this comment be removed without causing new eslint errors? if not, could you re-enable it after the comments block (as previously done line 87)